ISISComputingGroup/EPICS-rotating_sample_changer
# Ticket 4287: Update from tests

See https://github.com/ISISComputingGroup/IBEX/issues/4287

To test:
* Run the system tests
* Confirm the logic here matches that in original ticket
* Confirm documentation at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Sample-Changers looks correct
